_includes/manuals/2.2/1.2_release_notes.md

```diff
@@ -16,14 +12,252 @@ aren't familiar with all the methods Foreman provides. Documentation is availabl
+
+#### Users, Roles and Permissions
+* Error raised when Filter has a blank search condition ([#30244](https://projects.theforeman.org/issues/30244))
+* Impersonating disabled user is possible ([#30041](https://projects.theforeman.org/issues/30041))
+* Add ability to disable Users ([#28850](https://projects.theforeman.org/issues/28850))
+
+#### VM management
+* VNC console will not work if python3 is the default python. ([#30185](https://projects.theforeman.org/issues/30185))
+* VNC console for kubevirt ([#27319](https://projects.theforeman.org/issues/27319))
+
+#### Web Interface
+* Broken documentation link for 'RHV' in Compute Resource ([#30551](https://projects.theforeman.org/issues/30551))
+* Move user-dropdown to PF4 ([#30232](https://projects.theforeman.org/issues/30232))
+* User role with long name without spaces overlap to another column ([#30178](https://projects.theforeman.org/issues/30178))
+* UserDropdowns.scss is not used and is a duplicate ([#30139](https://projects.theforeman.org/issues/30139))
+* Host Detail Page Redesign  ([#30044](https://projects.theforeman.org/issues/30044))
+* follow naming convention for the auth sources tab ([#29958](https://projects.theforeman.org/issues/29958))
+* Bad name for numericInput component ([#29902](https://projects.theforeman.org/issues/29902))
+* Add cpu/cores/socket react cmponent ([#29900](https://projects.theforeman.org/issues/29900))
+* Update breadcrumbs to patternfly v4 ([#29332](https://projects.theforeman.org/issues/29332))
+* Update foreman fonts to patternfly v4 ([#29331](https://projects.theforeman.org/issues/29331))
+* ERF42-4995 - Invalid authenticity token (Session timed out at login screen) ([#10577](https://projects.theforeman.org/issues/10577))
+
+#### foreman-installer script
+* make it possible to brand katello-certs-check ([#30251](https://projects.theforeman.org/issues/30251))
+* Remove MongoDB Storage Engine Migration Hook ([#30006](https://projects.theforeman.org/issues/30006))
+* Remove the dependency on foreman-maintain ([#29963](https://projects.theforeman.org/issues/29963))
+* Drop --upgrade flag from katello hooks ([#29296](https://projects.theforeman.org/issues/29296))
+
+#### foreman-tasks
+* During smart proxy sync display the filter is set to "resource_id = 3 AND resource_type = SmartProxy", the tasks are displayed twice and Foreman Task id is set to "zero" ([#30376](https://projects.theforeman.org/issues/30376))
+
+*A full list of changes in 2.2.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1234)*
 
 ### Deprecations
 
 ### Upgrade warnings
 
-* Trends and statistics are now moved to separate plugin. See [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using it.
+* The `--upgrade` installer flag has been removed. When running the installer as part of the upgrade procedure it is not required.
```

Comment:
The `--upgrade` flag never existed in regular Foreman. This is just something that Katello used to do. Adding it here will only confuse Foreman users.